stacks/dars/openblas/README.md

```diff
@@ -14,4 +14,4 @@ docker build --no-cache -t clearlinux/stacks-dars-openblas .
 
 * `swupd_args` specifies [swupd update](https://github.com/clearlinux/swupd-client/blob/master/docs/swupd.1.rst#options) flags passed to the update during build.
 
->NOTE: An empty `swupd_args` will default to Clear Linux OS latest version. Consider this when building from the Dockerfile, as an OS update will be performed. The docker image in this registry was built and validated using version 29200.
+>NOTE: An empty `swupd_args` will default to Clear Linux OS latest version. Consider this when building from the Dockerfile, as an OS update will be performed. The docker image in this registry was built and validated using version 29480.
```

Comment:
@lsandoval please add to the release notes that CL base for this release is 29480.